ui/pages/confirmations/components/confirm/info/batch/batch-simulation-details/batch-simulation-details.tsx

```diff
@@ -54,10 +51,6 @@ export function BatchSimulationDetails() {
     [id, nestedTransactionIndexToEdit],
   );
 
-  if (transactionMeta?.type === TransactionType.revokeDelegation) {
```

Comment:
This was intentional to prevent an empty simulation row with `No estimated changes` on downgrade confirmations.

Related to the other comment explaining that an upgrade or downgrade can be an extension to a standard transaction.

Reply:
Fixed